can_messages.asciipb

```diff
@@ -293,17 +293,32 @@ msg {
 
 msg {
   id: 28
-  source: DRIVER_CONTROLS
-  # target: TELEMETRY
-  msg_name: "steering angle"
+  source: DRIVER_CONTROLS_STEERING
+  # target: DRIVER_CONTROLS_CENTER_CONSOLE
+  msg_name: "steering_output"
   can_data {
     u16 {
-      field_name_1: "steering_angle"
+      field_name_1: "control_stalk_analog_state"
+      field_name_2: "control_stalk_digital_state"
+    }
+  }
+}
+
+msg {
+  id: 29
+  source: DRIVER_CONTROLS_PEDAL
+  # target: DRIVER_CONTROLS_CENTER_CONSOLE
+  msg_name: "pedal_output"
+  can_data {
+    u16 {
+      field_name_1: "throttle"
+      field_name_2: "throttle_state"
```

Comment:
Is the idea that the `throttle_state` will indicate the zone that the pedal is in, doubling as an indicator of when the two ADC channel readings are not in agreement (ie. if the `throttle_state` isn't indicating a fault, then the `throttle` reading is good)?

Reply:
yeah that is correct